Project_source/app/src/main/java/android/clicker/school_live_simulator/StudyCourseState.kt

```diff
@@ -0,0 +1,22 @@
+package android.clicker.school_live_simulator
+
+abstract class StudyCourseState: TimerObserver {
+    /**
```

Comment:
Please, use same style everywhere.

Reply:
Should I only add curly brackets at functions?

Reply:
Done